Sources/MapboxNavigation/DayStyle.swift

```diff
@@ -155,6 +162,11 @@ open class DayStyle: Style {
         NavigationMapView.appearance().trafficUnknownColor = .trafficUnknown
         NavigationMapView.appearance().buildingDefaultColor = .defaultBuildingColor
         NavigationMapView.appearance().buildingHighlightColor = .defaultBuildingHighlightColor
+        NavigationMapView.appearance().routeDurationAnnotationColor = .routeDurationAnnotationColor
+        NavigationMapView.appearance().routeDurationAnnotationSelectedColor = .selectedRouteDurationAnnotationColor
+        NavigationMapView.appearance().routeDurationAnnotationFontName = "DIN Pro Medium"
```

Comment:
In general, when styling text on the map that we expect to render [server-side](https://docs.mapbox.com/ios/maps/api/6.3.0/customizing-fonts.html#server-side-fonts), we need to specify a list of fonts including a fallback font. Otherwise, the fontstack could be missing glyphs that are required for rendering the street names, even if the same street names render just fine on the base map: mapbox/mapbox-maps-ios#278.

Since font selection for a symbol layer is so specific to the style specification and unlike UIKit/Core Text font selection, the immediate goal should be to match the fonts in the base style.

Consider using the runtime styling API to get the `textFont` property of the symbol layer used to render roads (so the layer would have a source layer of `road`) and copying that expression to the layer for the annotations. Typically, a style will use multiple distinct fontstacks for various road label layers, but perhaps we could blithely pick the topmost matching layer or add heuristics to detect and ignore italics and boldface.

Reply:
These are good points. I changed the code to allow for a list of font names to be used and not just a single one. This lets the developer choose fonts that may be present in their map style. Also changed the default font of `DIN Pro Medium` to also include `Noto Sans CJK JP Medium` and `Arial Unicode MS Regular` as fallbacks in order to support more glyphs "out of the box" with our default set.
I think that the use of UIAppearance overrides for these map rendered fonts is probably the clearest way we can allow for supporting multiple fonts and fallbacks. I think it's preferable to trying to locate the font settings of a specific layer in the style, which feels more hidden and brittle, even if the source layer name for roads, `road` is unlikely to change very often in mapbox-streets.